src/app/shared/components/header/header.component.scss

```diff
@@ -2,6 +2,7 @@
   background-color: #343434;
   display: flex;
   flex-direction: row;
+  pointer-events: none;
```

Comment:
Why is this added?

Reply:
Just a mistake i believe, it has been removed.